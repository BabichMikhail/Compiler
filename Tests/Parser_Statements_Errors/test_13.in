var a, i : integer;
begin 
   	for i := 10 downto 1 begin
	 begin
		;;;; 
	
	a := 50;
end;
	end	
end.
