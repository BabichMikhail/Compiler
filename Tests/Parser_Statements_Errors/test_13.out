Fatal: Syntax error, "do" expected but "begin" found in Line 3 Column 29
